test/wasi/Makefile

```diff
@@ -12,5 +12,7 @@ wasm/pthread.wasm : c/pthread.c
 wasm/%.wasm : c/%.c
 	$(CC) $< $(CFLAGS) --target=$(TARGET) --sysroot=$(SYSROOT) -s -o $@
 
-.PHONY clean:
+.PHONY: clean
+.NOTPARALLEL: clean
```

Comment:
Do we need this `.NOTPARALLEL`?

> If the .NOTPARALLEL special target has prerequisites, then each of those prerequisites will be considered a target and all prerequisites of these targets will be run serially.

According to the doc, `.NOTPARALLEL` is only useful when the target in this list has prerequisites, and their prerequisites are run serially. `clean` has no prerequisites.